app/controllers/courses/age_range_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Courses
+  class AgeRangeController < ApplicationController
+    include EditBasicDetail
+
+    def edit
+      @options = @course.meta['edit_options']['age_range_in_years'].dup
+    end
+
+  private
+
+    def errors
+      params.dig(:course, :age_range_in_years) ? {} : { age_range_in_years: ["Pick an age range"] }
```

Comment:
Do we need to check that `course_age_range_in_years_other_to` and `course_age_range_in_years_other_from` are numbers?

Reply:
Validations being done as a separate story